### ISSUE TITLE
32 bit audio format has popping and clipping sound

### ISSUE THREAD

**Issue by Lordmyn** [nukeykt/Nuked-SC55#78]
Hi!

I use an external sound device, what is set to 32 Bit/96 kHz format. In this way the SC-55 emulator has popping and clipping sound. Everything will be fine if the bitrate is lowered to 24 or 16 bit.  Can it be corrected somehow?

I use Windows 10 Pro 22H2 x64.

Btw, the emulator is fantastic! I used to have a CM-500 module long time ago, I really love how the SC-55 mk I sounds (the mk II and the SC-88 and later models sounds very differently sometimes).

**Comment by Karmeck** [comment]
Maybe this, bit older now, version will work.
https://github.com/Falcosoft/Nuked-SC55/releases/tag/0.2.1

**Comment by Lordmyn** [comment]
Yes, I've tried this fork too with FP32 output, but surprisingly it has the same problem.

**Comment by johnnovak** [comment]
The good news is all you need is 16-bit output if you're just listening to it 😏
